docs/02_context.md

```
@@ -0,0 +1,691 @@
+    S->>S: Analyze all data
+    S-->>M: Return 500 token summary only
+    Note over M: Context: 65.5k not 145k!
+
+    M->>U: Present plan based on summary
+    Note over U: Sees descriptive plan
+
+    rect rgb(255, 200, 200)
+    Note over U,M: Problem: Original prompt too vague<br/>Should have specified imperative plan<br/>with exact file paths and signatures
+    end
+```
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)